modules/ROOT/partials/partial_devtools.adoc

```diff
@@ -115,7 +115,7 @@ With starknet-devnet-rs, you can do the following:
 link:https://github.com/0xSpaceShard/starknet-devnet-rs[The starknet-devnet-rs Github repository]
 
 
-[#katana]
+[#Katana]
```

Comment:
Please leave the id as-is. Id's impact links, so if you change an id, you have to check to make sure that there aren't any existing links to the id that also need to change.

This is not visible, except possibly in URLs, which we in any case want to leave as lower-case.